apps/minifront/src/state/swap/instant-swap.ts

```diff
@@ -31,6 +34,7 @@ import { divideAmounts } from '@penumbra-zone/types/amount';
 import { bech32mAssetId } from '@penumbra-zone/bech32m/passet';
 import { SwapSlice } from '.';
 import { sendSimulateTradeRequest } from './helpers';
+import { amountMoreThanBalance } from '../send';
```

Comment:
Since `amountMoreThanBalance` is now used by multiple slices, please extract it to the `./helpers.ts` file, so that slices aren't reaching into each other's code.